### DISCUSSION TITLE
Clarification on UseWebLogin

### DISCUSSION THREAD

**Discussion by PathToSharePoint** [pnp/powershell#275] [Q&A] [answered]
I haven't used PnP in a long time, so I apologize in advance if I missed a few steps.

I just installed the new version and tried to use it with UseWebLogin.

First, I am getting a warning that I should use "Interactive" instead, but the link doesn't help. What am I missing?
![image](https://user-images.githubusercontent.com/40323257/107889335-9c4ffd80-6ec6-11eb-8c07-bef82d969070.png)

Second, the templating commands return an access denied error ("code":"-2147024891). Is it to be expected? The commands work fine if I use Credentials for login.

Thanks!

**Comment by veronicageek** [comment, marked as answer]
We have a few articles that may help you catching up with the changes on the new PnP.PowerShell
https://pnp.github.io/powershell/articles/authentication.html

Also make sure to check the [CHANGELOG.md](https://github.com/pnp/powershell/blob/dev/CHANGELOG.md) for why we have the `-Interactive` parameter

**Comment by PathToSharePoint** [comment]
Thanks for the help!

Apparently one issue is that github URLs are case sensitive. So this would be the correct link:
https://pnp.github.io/powershell/cmdlets/Connect-PnPOnline.html#interactive-login-for-multi-factor-authentication

I'll try to locate the link and submit a PR.

**Comment by zdm378** [comment]
I'm following up on this since I don't see a PR out there and I'm new to github.
Regarding URL: https://pnp.github.io/powershell/cmdlets/Connect-PnPOnline.html

Issue is:
Current bug:
"WARNING: Consider using -Interactive instead, which provides better functionality. See the documentation at https://pnp.github.io/powershell/cmdlets/**connect-pnponline**.html#interactive-login-for-multi-factor-authentication"

vs.

Verified working:
"WARNING: Consider using -Interactive instead, which provides better functionality. See the documentation at https://pnp.github.io/powershell/cmdlets/**Connect-PnPOnline.**html#interactive-login-for-multi-factor-authentication"
Where the URL is case sensitive.

Also I suggested and edit to the requisite .MD page:
https://github.com/pnp/powershell/blob/dev/documentation/Connect-PnPOnline.md

![image](https://user-images.githubusercontent.com/79600887/109051460-2efb5400-76a0-11eb-8018-ef3a900c21eb.png)
Since it is also using a lowercase URL.
